client/src/main/java/com/aliyun/emr/rss/client/read/RetryingChunkClient.java

```diff
@@ -217,7 +217,10 @@ private synchronized boolean shouldRetry(Throwable e) {
             || e instanceof TimeoutException
             || (e.getCause() != null && e.getCause() instanceof TimeoutException)
             || (e.getCause() != null && e.getCause() instanceof IOException);
-    return isIOException && hasRemainingRetries();
+    boolean isSortException =
```

Comment:
We can let all new added Exceptions inherit from IOException, so we needn't change here.

Reply:
> We can let all new added Exceptions inherit from IOException, so we needn't change here.

Nice suggestion!